modules/social_features/social_activity/social_activity.module

```diff
@@ -23,6 +23,12 @@ function social_activity_social_user_account_header_items(array $context) {
   $account = $context['user'];
 
   $notifications_view = views_embed_view('activity_stream_notifications', 'block_1');
+  // Mark all notifications as seen.
+  $notifications_view['#attached'] = [
+    'library' => [
```

Comment:
This could theoretically break if the view already has a library or other attachment.

Can do something like (untested):
```php
if (!isset($notifications_view['#attached']) {
  $notifications_view['#attached'] = [];
}
if (!isset($notifications_view['#attached']['library']) {
  $notifications_view['#attached']['library'] = [];
}
$notifications_view['#attached']['library'][] = 'activity_creator/activity_creator.notifications';
```

Reply:
It's more verbose but it it's also more robust.